views/drones/update-form.hbs

```diff
@@ -1 +1,39 @@
-<h2>Update the drone</h2>
\ No newline at end of file
+<div class="title"></div>
+<h2>Update the drone</h2>
+</div>
+<hr />
+
+
```

Comment:
hey! The update form is not working because the button is outside the form tag 😅